src/main/java/eu/europeana/iiif/service/ManifestService.java

```diff
@@ -232,11 +229,17 @@ public Boolean existsFullText(String fullTextUrl) throws IIIFException {
                 } else if (responseCode == HttpStatus.SC_OK) {
                     result = Boolean.TRUE;
                 } else {
-                    throw new FullTextCheckException("Error checking if full text exists: "+response.getStatusLine().getReasonPhrase());
+                    // TODO when hysterix is enabled again, we can simply throw an error again
+                    //throw new FullTextCheckException("Error checking if full text exists: "+response.getStatusLine().getReasonPhrase());
+                    LOG.error(String.format("Error checking if full text exists: %s", response.getStatusLine().getReasonPhrase()));
```

Comment:
No need to use String.format, this is built-in to log4j2. Better use 
LOG.error("Error checking if full text exists: {}", response.getStatusLine().getReasonPhrase())